src/EditorFeatures/CSharpTest/UseSimpleUsingStatement/UseSimpleUsingStatementTests.cs

```diff
@@ -0,0 +1,535 @@
+        {
+            Console.WriteLine(a);
+        }
+    }
+}",
+@"using System;
+
+class C
+{
+    void M()
+    {
+        using var a = b;
+        using var c = d;
+        Console.WriteLine(a);
+    }
+}",
+parseOptions: CSharp8ParseOptions);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseSimpleUsingStatement)]
+        public async Task TestMultiUsingOnlyOnTopmostUsing()
+        {
+            await TestMissingAsync(
+@"using System;
+
+class C
+{
+    void M()
+    {
+        using (var a = b)
+        [||]using (var c = d)
+        {
+            Console.WriteLine(a);
+        }
+    }
+}",
+new TestParameters(parseOptions: CSharp8ParseOptions));
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseSimpleUsingStatement)]
+        public async Task TestFixAll1()
+        {
+            await TestInRegularAndScriptAsync(
+@"using System;
+
+class C
+{
+    void M()
+    {
+        {|FixAllInDocument:|}using (var a = b)
+        {
+            using (var c = d)
+            {
+                Console.WriteLine(a);
+            }
+        }
+    }
+}",
+@"using System;
+
+class C
+{
+    void M()
+    {
+        using var a = b;
+        using var c = d;
+        Console.WriteLine(a);
+    }
+}",
+parseOptions: CSharp8ParseOptions);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseSimpleUsingStatement)]
+        public async Task TestFixAll2()
+        {
+            await TestInRegularAndScriptAsync(
+@"using System;
+
+class C
+{
+    void M()
+    {
+        using (var a = b)
+        {
+            {|FixAllInDocument:|}using (var c = d)
+            {
+                Console.WriteLine(a);
+            }
+        }
+    }
+}",
+@"using System;
+
+class C
+{
+    void M()
+    {
+        using var a = b;
+        using var c = d;
+        Console.WriteLine(a);
+    }
+}",
+parseOptions: CSharp8ParseOptions);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseSimpleUsingStatement)]
+        public async Task TestFixAll3()
+        {
+            await TestInRegularAndScriptAsync(
+@"using System;
+
+class C
+{
+    void M()
+    {
+        {|FixAllInDocument:|}using (var a = b)
+        using (var c = d)
+        {
+            using (var e = f)
+            using (var g = h)
+            {
+                Console.WriteLine(a);
+            }
+        }
+    }
+}",
+@"using System;
+
+class C
+{
+    void M()
+    {
+        using var a = b;
+        using var c = d;
+        using var e = f;
+        using var g = h;
+        Console.WriteLine(a);
+    }
+}",
+parseOptions: CSharp8ParseOptions);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsUseSimpleUsingStatement)]
+        public async Task TestFixAll4()
+        {
+            await TestInRegularAndScriptAsync(
+@"using System;
+
+class C
+{
+    void M()
+    {
+        {|FixAllInDocument:|}using (var a = b)
```

Comment:
All the FixAll tests involve nesting. Would it make sense to test two sibling `using` statements too?
```C#
using (...) { }
using (...) { }
```